lighthouse-core/report/v2/renderer/dom.js

```diff
@@ -98,6 +97,21 @@ class DOM {
   document() {
     return this._document;
   }
+
+  /**
+   * Guaranteed context.querySelector. Always returns an element or throws if
+   * nothing matches query.
+   * @param {!DocumentFragment|!Element} context
+   * @param {string} query
+   * @return {!Element}
+   */
+  static find(context, query) {
```

Comment:
to be more like https://www.w3.org/TR/selectors-api2/ it would make more sense to do `find(query, context)`.
i'd prefer it, as well.